src/.gitignore

```diff
@@ -0,0 +1,23 @@
+.DS_Store
```

Comment:
Can we move all these files to a directory called 'frontend'?

Reply:
U mean, change the folder SRC to Frontend?

Reply:
No, I just mean create a folder called 'frontend' at the root, and move all of these files including the src folder into it.